CHANGELOG.md

```diff
@@ -3,6 +3,68 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.11.1
+
+_Aug 11, 2023_
+
+We'd like to offer a big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- 💫 Add theme automentation for the Tree View component
```

Comment:
```suggestion
- 💫 Add theme augmentation for the Tree View component
```

Reply:
Or this, because it's the whole package that had the `themeAugmentation` added and it also contains augmentation for the `TreeItem` component.
```suggestion
- 💫 Add theme augmentation to `@mui/x-tree-view`
```